Tasks/Common/azure-arm-rest/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -0,0 +1,29 @@
+{
+  "loc.messages.ClientIdCannotBeEmpty": "clientId must be a non empty string.",
+  "loc.messages.DomainCannotBeEmpty": "domain must be a non empty string.",
+  "loc.messages.SecretCannotBeEmpty": "secret must be a non empty string.",
+  "loc.messages.armUrlCannotBeEmpty": "arm Url must be a non empty string.",
+  "loc.messages.authorityUrlCannotBeEmpty": "authority must be a non empty string.",
+  "loc.messages.CouldNotFetchAccessTokenforAzureStatusCode": "Could not fetch access token for azure. Status code: %s, status message: %s",
+  "loc.messages.CallbackCannotBeNull": "callback cannot be null.",
+  "loc.messages.VMNameCannotBeNull": "vmName cannot be null or undefined and it must be of type string.",
+  "loc.messages.InvalidValue": "%s is not a valid value. The valid values are: %s",
+  "loc.messages.VmExtensionNameCannotBeNull": "vmExtensionName cannot be null or undefined and it must be of type string.",
+  "loc.messages.ExpandShouldBeOfTypeString": "expand must be of type string.",
+  "loc.messages.ExtensionParametersCannotBeNull": "extensionParameters cannot be null or undefined.",
+  "loc.messages.LoadBalancerNameCannotBeNull": "'loadBalancerName cannot be null or undefined and it must be of type string.'",
+  "loc.messages.ParametersCannotBeNull": "parameters cannot be null or undefined.",
+  "loc.messages.NetworkInterfaceNameCannotBeNull": "networkInterfaceName cannot be null or undefined and it must be of type string.",
+  "loc.messages.NetworkSecurityGroupNameCannotBeNull": "networkSecurityGroupName cannot be null or undefined and it must be of type string.",
+  "loc.messages.SecurityRuleNameCannotBeNull": "securityRuleName cannot be null or undefined and it must be of type string.",
+  "loc.messages.SecurityRuleParametersCannotBeNull": "securityRuleParameters cannot be null or undefined.",
+  "loc.messages.DeploymentNameCannotBeNull": "deploymentName cannot be null or undefined and it must be of type string.",
+  "loc.messages.CredentialsCannotBeNull": "'credentials' cannot be null.",
+  "loc.messages.SubscriptionIdCannotBeNull": "'subscriptionId' cannot be null.",
+  "loc.messages.InvalidResponseLongRunningOperation": "Invalid response received for fetching status of a long running operation.",
+  "loc.messages.TimeoutWhileWaiting": "Timed out while waiting",
+  "loc.messages.ResourceGroupCannotBeNull": "resourceGroupName cannot be null or undefined and it must be of type string.",
+  "loc.messages.ResourceGroupExceededLength": "\"resourceGroupName\" should satisfy the constraint - \"MaxLength\": 90",
+  "loc.messages.ResourceGroupDeceededLength": "\"resourceGroupName\" should satisfy the constraint - \"MinLength\": 1",
+  "loc.messages.ResourceGroupDoesntMatchPattern": "\"resourceGroupName\" should satisfy the constraint - \"Pattern\": /^[-\\w\\._\\(\\)]+$/"
+}
```

Comment:
As per other common module, this should work. I will test after removing strings from task.